lib/go-tc/profiles.go

```diff
@@ -91,6 +92,28 @@ type Profile struct {
 	Parameters      []ParameterNullable `json:"params,omitempty"`
 }
 
+// ProfilesResponseV5 is a list of profiles returned by GET requests.
+type ProfilesResponseV5 struct {
+	Response []ProfileV5 `json:"response"`
+	Alerts
+}
+
+// A ProfileV5 represents a set of configuration for a server or Delivery Service
+// which may be reused to allow sharing configuration across the objects to
+// which it is assigned. Note: Field LastUpdated represents RFC3339
+type ProfileV5 struct {
+	ID              int                 `json:"id" db:"id"`
+	LastUpdated     time.Time           `json:"lastUpdated" db:"last_updated"`
+	Name            string              `json:"name" db:"name"`
+	Parameter       string              `json:"param"`
```

Comment:
What is this field used for?

Reply:
A whole bunch of test cases.. But great question to ask.. I had it removed and then I saw massive failures in tests.

Reply:
But... that doesn't make any sense. You just added the structure, and it didn't exist in the previous version, `ProfileNullable`

Reply:
It didn't in this `ProfileNullable` but it exists in `Profile`

Reply:
`Profile` isn't used in the API though, only `ProfileNullable` is, so the real previous version of the struct is the latter.